tests/pxp-module-puppet.rb

```diff
@@ -0,0 +1,14 @@
+require 'facter'
```

Comment:
It was something I was using to generate/test the template. I can drop it out again, but it has no functional impact.